src/OrbitGl/ScopeTree.h

```diff
@@ -25,7 +25,9 @@ template <typename ScopeT>
 class ScopeNode {
  public:
   ScopeNode() = default;
```

Comment:
children_by_start_time_ is going to be nullptr after default c-tor, is this intended?

Reply:
Fixed by using non-static data member initialization.